partials/language-specific-deploy/docker.md

```diff
@@ -4,7 +4,7 @@
 Be sure that you:
 
 * push on the **master branch**.
-* have and commit a file named **Dockerfile**, [Here is what it will look like](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices "Dockerfile")
+* have and commit a file named **Dockerfile** (Or use the **CC_DOCKERFILE** environment variable if your Dockerfile has a different name), [Here is what it will look like](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices "Dockerfile")
```

Comment:
```suggestion
* have and commit a file named **Dockerfile** (Or use the **CC_DOCKERFILE** environment variable if your Dockerfile has a different name), [Here is what it will look like](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/ "Dockerfile")
```

I also think that it would be better to say `have and commit a file named Dockerfile or use the CC_DOCKERFILE + "link to references variable" if your docker file has a different name`